army.cpp

```diff
@@ -1134,9 +1162,24 @@ int Army::RemoveEffects(int num, char const *effect)
 	return(ret);
 }
 
+WeaponBonusMalus* GetWeaponBonusMalus(WeaponType *weapon, WeaponType *target) {
```

Comment:
i am 100% in favor of multi hit weapons but weapon bonus towards another weapon imho complicates things. Can you please extract it into separated PR as i would love to have multi damage weapons but still discuss this.

Reply:
I will not have time to extract this into separate PR. Just do not add any bonuses. Also, there are missing methods to add bonuses for the ruleset, I'll add them later.